scheduler/remote_server.py

```diff
@@ -388,27 +404,22 @@ def calculate_new_last_line_params():
             inf_dur_line['time'] = queued_finish
             queued_lines.append(inf_dur_line)
 
-    queued_dict = collections.OrderedDict()
-    for line in queued_lines:
-        if not line['order'] in queued_dict:
-            queued_dict[line['order']] = []
-
-        queued_dict[line['order']].append(line)
-
-    return queued_dict, queued_lines
+    return queued_lines, warnings
 
 def timeline_to_atq(timeline, scd_dir, time_of_interest):
     """ Converts the created timeline to actual atq commands.
 
     Args:
-        timeline (OrderedDict): A dictionary holding all timeline events.
+        timeline (List): A list holding all timeline events.
         scd_dir (str): The directory with SCD files.
         time_of_interest (datetime): The datetime holding the time of scheduling.
 
     Log and backup the existing atq, remove old events and then schedule everything recent. The
     first entry should be the currently running event, so it gets scheduled immediately. This
     function only backs up the commands that have not run yet.
     """
+    # Convert to ordered dict
+    timeline = timeline_to_dict(timeline)
```

Comment:
I don't think you need this line. Instead you could just change line 447/448 to 'for event in timeline:' @kkotyk please confirm

Reply:
Changed this